predict.py

```diff
@@ -63,8 +63,8 @@ def dump(path, data):
 config = state_dict['config']
 config["cuda"] = args.cuda
 model = MTDNNModel(config, state_dict=state_dict)
-
-test_metrics, test_predictions, scores, golds, test_ids = eval_model(model, test_data,
+with torch.no_grad():
```

Comment:
Should put this in prediction in model.py?  This has less changes.

Reply:
less changes but the `predict` function will be called once for every batch?

Reply:
Good point.